notebooks/data-acquisition/03-download-sliiders-input-data.ipynb

```diff
@@ -527,12 +527,7 @@
     "with sset.PATH_GWDB_RAW.open(\"wb\") as f:\n",
     "    f.write(\n",
     "        requests.get(\n",
-    "            \"https://www.ubs.com/global/en/family-office-uhnw/reports/\"\n",
-    "            \"global-wealth-report-2023/_jcr_content/mainpar/toplevelgrid_5684475/col2/\"\n",
-    "            \"linklistnewlook/link_copy.0357374027.file/\"\n",
-    "            \"PS9jb250ZW50L2RhbS9hc3NldHMvd20vZ2xvYmFsL2ltZy9nbG9iYWwtZmFtaWx5LW9mZmljZS\"\n",
-    "            \"9kb2NzL2RhdGFib29rLWdsb2JhbC13ZWFsdGgtcmVwb3J0LTIwMjMtZW4ucGRm/\"\n",
-    "            \"databook-global-wealth-report-2023-en.pdf\"\n",
+    "            \"https://urldefense.us/v3/__https:/www.ubs.com/global/en/family-office-uhnw/reports/global-wealth-report-2023/_jcr_content/mainpar/toplevelgrid_5684475_1708633751/col1/innergrid/xcol1/actionbutton_copy_co.1784379955.file/PS9jb250ZW50L2RhbS9hc3NldHMvd20vZ2xvYmFsL2ltZy9nbG9iYWwtZmFtaWx5LW9mZmljZS9kb2NzL2d3ci0yMDIzLWVuLTIucGRm/gwr-2023-en-2.pdf__;!!PvBDto6Hs4WbVuu7!LkcjRnpeJpav7n7pccnu9GARkLWeYSdBrUR7_2wbXi9tKQiH0E3ypwI9h8cm0uHmnY0eFJ3da07dQQScVsUHry39Hg$\"\n",
```

Comment:
I just checked this link and it looks like it's not correct - sorry for sending you the wrong one. If you use this link, you'd see that it goes to the "report" rather than the "databook". Here's a link to the databook:

https://rev01ution.red/wp-content/uploads/2024/03/global-wealth-databook-2023-ubs.pdf

Also, the "urldefense" part of the URL is just something that comes through on email clients so can be removed. But let's use the link above, otherwise it will be downloading the wrong thing (which is why you didn't see Table 2-1 or Table 2-2 when you looked at it)